docs/how-to/programming_guide.rst

```diff
@@ -0,0 +1,37 @@
+.. meta::
+    :description: Programming guide
+    :keywords: HIP, programming guide, heterogeneous programming, AMD GPU programming
+
+.. _hip-programming-guide:
+
+********************************************************************************
+Programming guide
+********************************************************************************
+
+ROCm provides a robust environment for heterogeneous programs running on CPUs
+and AMD GPUs. ROCm supports a variety of programming languages and frameworks to
+help developers access the power of AMD GPUs. The natively supported programming
+languages are HIP (Heterogeneous-Compute Interface for Portability) and
+OpenCL, but HIP bindings are available for Python and Fortran. 
```

Comment:
```suggestion
ROCm provides a robust environment for heterogeneous programs running on CPUs
and AMD GPUs. ROCm supports various programming languages and frameworks to
help developers access the power of AMD GPUs. The natively supported programming
languages are HIP (Heterogeneous-Compute Interface for Portability) and
OpenCL, but HIP bindings are available for Python and Fortran. 
```

Reply:
And should it be "programs running on CPUs and AMD GPUs" or "programs running on AMD CPUs and GPUs"?

Reply:
programs running on CPUs and AMD GPUs